documentation/css/style.css

```diff
@@ -210,6 +210,10 @@
     height: 20rem;
 }
 
+.slider-height{
+    height: 9rem;
+}
+
 .container-input{
     display: flex;
```

Comment:
can be clubbed into a class, as it is used at various places

Reply:
Sure